lib/sparql.jison

```diff
@@ -642,7 +642,7 @@ SelectQuery
     | SelectClauseVars     DatasetClause* WhereClause SolutionModifier
     {
       // Check for projection of ungrouped variable
-      if (!Parser.skipUngroupedVariableCheck) {
+      if (!Parser.skipValidation) {
```

Comment:
```suggestion
      if (!Parser.skipUngroupedVariableCheck && !Parser.skipValidation) {
```

so we maintain compatibility for the entire major version.

Reply:
I fixed it in sparql.js, that way the parser code looks cleaner to me. Would this be a good way or would you rather have it in the way you have suggested?